docker-compose.yml

```diff
@@ -101,7 +101,7 @@ services:
       - ipfs
     command: >
```

Comment:
Even better would be to replace this with the wait-for-it script as is done in frost-api. :-)

Reply:
That would be nice, but maybe merge this one as is and use wait-for-it in another PR if this unblocks something...?

Reply:
If someone is blocked, they should use a `docker-compose.override.yml` to adjust the command for their particular situation.